beacon-chain/rpc/apimiddleware/endpoint_factory.go

```diff
@@ -258,6 +259,13 @@ func (f *BeaconEndpointFactory) Create(path string) (*gateway.Endpoint, error) {
 			GetResponse:        &produceBlockResponseJson{},
 			RequestURLLiterals: []string{"slot"},
 			RequestQueryParams: []gateway.QueryParam{{Name: "randao_reveal", Hex: true}, {Name: "graffiti", Hex: true}},
+			Err:                &gateway.DefaultErrorJson{},
```

Comment:
Missed this in https://github.com/prysmaticlabs/prysm/pull/9259#discussion_r675369854